WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Connection Alerts/PointOfSaleCardPresentPaymentFoundMultipleReadersView.swift

```diff
@@ -13,21 +13,18 @@ struct PointOfSaleCardPresentPaymentFoundMultipleReadersView: View {
     }
 
     var body: some View {
-        VStack(spacing: 0) {
+        VStack {
             Text(Localization.headline)
                 .font(.posTitleEmphasized)
                 .padding(Layout.headerPadding)
                 .accessibilityAddTraits(.isHeader)
 
-            List(readerIDs, id: \.self) { readerID in
-                VStack {
-                    readerRow(readerID: readerID)
+            scanningText()
```

Comment:
Much nicer to have the scanning text up here. It also makes the UI less twitchy when the number of readers found changes 💯